doc/compiling/windows.md

```diff
@@ -2,67 +2,82 @@
 
 ## Requirements
 
-- [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
-- [CMake](https://cmake.org/download/)
-- [vcpkg](https://github.com/Microsoft/vcpkg)
-- [Git](https://git-scm.com/downloads)
-
+-   [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
+-   [CMake](https://cmake.org/download/)
+-   [vcpkg](https://github.com/Microsoft/vcpkg)
+-   [Git](https://git-scm.com/downloads)
 
 ## Compiling and installing the dependencies
 
-It is highly recommended to use vcpkg as package manager.
+It is highly recommended to use vcpkg as package manager, installed in a location without spaces in the path for compatibility with CMake.
```

Comment:
```suggestion
It is highly recommended to use vcpkg to manage dependencies, installed at a path without spaces for compatibility with the Luanti build configuration.
```

Reply:
CMake handles spaces fine. It's our fault spaces don't work, although anybody using spaces should know better than to test us.

Reply:
"path without spaces *in it*" sounds a little better to me

Reply:
```suggestion
It is highly recommended to use vcpkg to manage dependencies, installed at a path without spaces in it for compatibility with the Luanti build configuration.
```